domain/src/main/java/uk/gov/moj/sdt/domain/RequestType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.moj.sdt.domain;
+
+public enum RequestType {
+
+    BREATHINGSPACE("mcolBreathingSpace"),
```

Comment:
Use underscored names across all the variables for consistency. BREATHING_SPACE, DEFENCE_CRITERIA

Reply:
done..